src/clarity-angular/modal/modal.ts

```diff
@@ -92,7 +94,7 @@ export class Modal implements OnChanges, OnDestroy {
     @Input("clrModalPreventClose") stopClose: boolean = false;
     @Output("clrModalAlternateClose") altClose: EventEmitter<boolean> = new EventEmitter<boolean>(false);
 
-    constructor(private _scrollingService: ScrollingService) {}
+    constructor(private _scrollingService: ScrollingService, private elementRef: ElementRef) {}
```

Comment:
Thanks for making the changes. I believe we can remove the ElementRef now since its not being used.